pelias/api
# Convert endpoint

`/v1/convert` endpoint accepts the following REST parameters:

`from`: Origin coordinate type. Currently only `mgrs` is supported .
`to`: Destination coordinate type. Currently only `decdeg` is supported. Converts origin to geodetic coordinates in WGS84 datum.
`q`: Coordinate itself.

This endpoint acts as a proxy for the [Geotrans MGRS Converter](https://github.com/venicegeo/geotrans-mgrs-converter) Docker container serving at port 3150.

![image](https://user-images.githubusercontent.com/32872347/34961890-4547b3ea-fa0f-11e7-8252-12e6ab6e0ed6.png)

**Instructions to test:**
1. Run [Geotrans MGRS Converter](https://github.com/venicegeo/geotrans-mgrs-converter) dockerfile. Ensure that converter is being server at port 3150.
2. Run this Pelias API. Test converter with `from`, `to`, and `q` parameters populated.
